### PR TITLE
#75941936 Created blog routes

### PR DESCRIPTION
**Pull request by Procech20**
>What does the PR do?

Creates routes for the blogs with all CRUD functionalities 

>What is included?

GET        api/v1/techblogs
GET        api/v1/techblogs/:id
POST     api/v1/techblogs
PUT       api/v1/techblogs/:id
DELETE api/v1/techblogs/:id

All routes are validated as required

### CODE REVIEW

File: src/App/middlewares/validator.js
Comment (RedJanvier) [inline src/App/middlewares/validator.js:1]: ⚠️ THIS IS NOT A SERVICE!!!!!!!
Please make sure you move it for your next PR.

**Review by RedJanvier — APPROVED** [review]
Make sure to fix the comments left and wait for a review from @Code-250

**Review by Code-250 — APPROVED** [review]

File: src/App/middlewares/validator.js
Comment (Procech20) [reply src/App/middlewares/validator.js:1]: Actually  already have a validator middleware this is a schema service

**Review by Code-250 — APPROVED** [review]
hello so far you can merge this PR

**Pull request merged**
